src/webapi/xml_http_request.rs

```diff
@@ -0,0 +1,69 @@
+use webcore::value::{
+    Reference,
+    Value,
+};
+use webcore::try_from::TryInto;
+
+/// XMLHttpRequest object to make http requests
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest)
+#[derive(Debug)]
+pub struct XMLHttpRequest {
+    reference: Reference,
+}
```

Comment:
Please make this the same as other `Reference` wrappers. (e.g. look at `src/webapi/file.rs` for a minimal example) In this case the `XMLHttpRequest` derives from `IEventTarget`.